src/interfaces/filing-interfaces/filing-interfaces.ts

```diff
@@ -78,6 +78,7 @@ export interface RegistrationFilingIF {
     effectiveDate?: string // not saved by UI but may be provided by API
     filingId?: number // not saved by UI but may be provided by API
     folioNumber?: string // optional and only displayed for certain account types
+    isTransactionalFolioNumber?: boolean // Optional - applies to registration Premium accounts only
```

Comment:
According to filing-template-mixin below, this property is used for dissolution premium accounts also.

Reply:
This property is already implemented for dissolution premium accounts. 

> https://github.com/bcgov/business-create-ui/blob/cf94573b2ae02982b643acd10123380019000832/src/interfaces/filing-interfaces.ts#L102

Reply:
Oh, right 😊 

Then it's redundant to say it only applies to one type of filing when these objects are already specific to that filing type 😉